projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -122,12 +120,14 @@ const ProductCard = props => {
 	const isInactive = status === PRODUCT_STATUSES.INACTIVE;
 	const isAbsent = status === PRODUCT_STATUSES.ABSENT || status === PRODUCT_STATUSES.NEEDS_PURCHASE;
 	const isPurchaseRequired = status === PRODUCT_STATUSES.NEEDS_PURCHASE;
+	const isLinkCard = isPurchaseRequired || isAbsent;
```

Comment:
`isAbsent` is not the same as this? It looks exactly if the status is absent or needs purchase.